apps/test/unit/templates/instructions/TeacherFeedbackTest.jsx

```diff
@@ -111,6 +113,56 @@ describe('TeacherFeedback', () => {
   });
 
   describe('viewed as Teacher', () => {
+    it('displays correct message with checkmark if student has viewed their feedback', () => {
+      const props = {
+        ...TEACHER_FEEDBACK_NO_RUBRIC_PROPS,
+        latestFeedback: [
+          {feedback_provider_id: 5, student_seen_feedback: new Date()}
+        ]
+      };
+      const seenByStudentSpy = sinon.spy(i18n, 'seenByStudent');
+      const wrapper = shallow(<TeacherFeedback {...props} />);
+      expect(wrapper.find('FontAwesome').length === 1).to.equal(true);
```

Comment:
NIT: (just an fyi) this can also be written as `expect(wrapper.find('FontAwesome')).to.have.lengthOf(1)`